Sources/Plasma/Apps/plClient/Mac-Cocoa/main.mm

```diff
@@ -580,6 +601,10 @@ - (NSApplicationPresentationOptions)window:(NSWindow*)window
            NSApplicationPresentationAutoHideMenuBar;
 }
 
+- (void)windowDidChangeScreen:(NSNotification *)notification
+{
+}
+
```

Comment:
needed?

Reply:
I'm leaving this in place because it will be. There's going to have to be code to wired in to fix multiple displays - and this will be an important bit.

But thats a deeper set of changes that's going to affect both Mac and Windows.